src/main/java/games/strategy/engine/framework/startup/mc/GameSelectorModel.java

```diff
@@ -25,40 +25,38 @@
 import games.strategy.triplea.ai.proAI.ProAI;
 import games.strategy.triplea.settings.ClientSetting;
 
+/**
+ * Data model that represents the users current 'game' selection.
+ * This includes for example which file was used to load the game,
+ * a partially parsed game data object.
+ */
 public class GameSelectorModel extends Observable {
-  /**
-   * Returns the name of the directory within a map's directory where the game xml is held.
-   * Example: returns "games" which would be the games folder of "triplea/maps/someMapFooBar/games"
-   */
-  public static final String DEFAULT_GAME_XML_DIRECTORY_NAME = "games";
-  private GameData m_data = null;
-  private String m_gameName = "";
-  private String m_gameVersion = "";
-  private String m_gameRound = "";
-  private String m_fileName = "";
-  private boolean m_canSelect = true;
-  private boolean m_isHostHeadlessBot = false;
+  private GameData data;
+  private String gameName = "";
+  private String gameVersion = "";
+  private String gameRound = "";
+  private String fileName = "";
+  private boolean canSelect = true;
+  private boolean isHostHeadlessBot = false;
   // just for host bots, so we can get the actions for loading/saving games on the bots
   // from this model
-  private ClientModel m_clientModelForHostBots = null;
+  private ClientModel clientModelForHostBots;
 
   public GameSelectorModel() {
     setGameData(null);
-    m_fileName = "";
   }
 
   public void resetGameDataToNull() {
     setGameData(null);
-    m_fileName = "";
```

Comment:
Is this statement safe to remove?  It doesn't appear that `setGameData()` resets `fileName` to an empty string.

Reply:
Good catch, likely not, I think I thought this was a constructor.